src/modeler/AutoRest.Swagger.Tests/Resource/Swagger/Validation/positive/property-names-casing-valid.json

```diff
@@ -0,0 +1,121 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "Property names should be camelCase style",
+    "description": "Some documentation.",
+    "version": "2017-02-08"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/",
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "paths": {
+    "/foo": {
+      "put": {
+        "operationId": "PutFoo",
+        "summary": "Foo path",
+        "description": "Foo operation",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
```

Comment:
@dsgouda was saying that the linter should have flagged for Anonymous properties. He was surprised at it not being flagged.